fmn/fmn_fasshim.py

```diff
@@ -23,11 +24,28 @@
 default_url = 'https://admin.fedoraproject.org/accounts/'
 creds = config.app_conf['fas_credentials']
 
-fasclient = fedora.client.fas2.AccountSystem(
-    base_url=creds.get('base_url', default_url),
-    username=creds['username'],
-    password=creds['password'],
-)
+if config.get('fasjson'):
+    client = fasjson_client.Client(url=creds.get('base_url', default_url))
+else:
+    client = fedora.client.fas2.AccountSystem(
+        base_url=creds.get('base_url', default_url),
+        username=creds['username'],
+        password=creds['password'],
+    )
+
+def make_fasjson_cache(**config):
+    log.warn("Building the FASJSON cache into redis.")
+    if _cache.get('fas_cache_built'):
+        log.warn("FASJSON cache already built into redis.")
+        return
+    global client
+    try:
+        _add_to_cache(list(client.list_all_entities("users")))
+    except fasjson_client.errors.APIError as e:
+        log.error("Something went wrong building cache with error: %s" % e)
+        continue
```

Comment:
Flake8 also caught that you can't have a `continue` statement outside of a loop, it should be `return`.